src/colorizer/DataCache.ts

```diff
@@ -124,13 +188,14 @@ export default class DataCache<E extends DisposableValue | Object> {
   public get(key: string | number): E | undefined {
```

Comment:
Should the whole class, in addition to being generic over its element type `E`, be generic over some `K extends string | number` so that instances can narrow keys to only `number`s or `string`s?

Reply:
Made this change!